### ISSUE TITLE
Ticking entity (AI crash on dedicated server)

### ISSUE THREAD

**Issue by Maokuma** [JurassiCraft/JurassiCraft#167]
Hi, I upgraded to build "JurassiCraft-1.3.2-build-100-universal" and as soon as a dino spawn's or get loaded it crashes the server with this crash-log: http://pastebin.com/uBXXU9af
I'm running it on an dedicated server with pure forge 1291.
The crash triggers in an instant and is easy to replicate. And singelplayer works fine.

I don't know how the AI code works so I'll try to see if it only affects some dinosaurs and post some extra info if there is a change. 
//Dilophosauruses are scary

**Comment by Kash-117** [comment]
Maybe you should uninstall the mod and reinstall it

**Comment by Maokuma** [comment]
Thanks Kash-117. I have tested some stuff but no mater except single player, as soon as a dino spawns, it crashes the game. That means running the mod with a new clean world and no extra mods.

**Comment by Maokuma** [comment]
I have tested most of the dinos now against an dedicated forge server with only Jurassicraft and NEI installed and they all generate the same error: (java.lang.NoSuchMethodError: net.ilexiconn.jurassicraft.entity.EntityJurassiCraftCreature.func_70666_h(F)Lnet/minecraft/util/Vec3;).

I also tested build version 101 but it wont spawn or hatch any dinos except in single player.

//As always, thanks for your work, looking forward to the new changes to the dinos.
